torch_xla/csrc/ops/native_dropout.cpp

```diff
@@ -0,0 +1,36 @@
+#include "torch_xla/csrc/ops/native_dropout.h"
+
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input) {
+  xla::Shape input_shape = GetXlaShape(input);
+  return xla::ShapeUtil::MakeTupleShape({input_shape, input_shape});
+}
+
+}  // namespace
+
+NativeDropout::NativeDropout(const torch::lazy::Value& input, float p,
+                             c10::optional<bool> train,
+                             const torch::lazy::Value& seed)
+    : XlaNode(torch::lazy::OpKind(at::aten::native_dropout), {input, seed},
+              [&]() { return NodeOutputShape(input); }, 2),
+      p_(p),
+      train_(train) {}
+
+torch::lazy::NodePtr NativeDropout::Clone(torch::lazy::OpList operands) const {
+  return torch::lazy::MakeNode<NativeDropout>(operands.at(0), operands.at(1),
```

Comment:
`operands` should only contain Lazy IR Node. So here it should be:
```
return torch::lazy::MakeNode<NativeDropout>(operands.at(0), p_, train_);
```

Reply:
Should we also consider seed (with type torch::lazy::Value) as a lazy IR node?

Reply:
I just checked example multinomial.cpp, the seed is also considered as the lazy IR node. Also fixed p_ and train_ to be val. Thanks!